lib/portlayer/exec/container_cache.go

```diff
@@ -32,45 +35,41 @@ type containerCache struct {
 	cache map[string]*Container
 }
 
-var containers *containerCache
+var Containers *containerCache
```

Comment:
Yes, but we do initialize this in exec.Init(). I will move this to package init() when we have a new package for the container cache.